### PR TITLE
Allow selecting all things from all pages in a table

### PR DESCRIPTION
**Pull request by apmasell**
JIRA: GLT-2970

### CODE REVIEW

File: miso-web/src/main/webapp/scripts/list.js [hunk 1]
```diff
@@ -298,6 +298,29 @@ ListUtils = (function($) {
           updateSelectedLabel(state);
         }
       });
+      staticActions.push({
+        "name": "🗹",
```
Comment (avarsava) [inline miso-web/src/main/webapp/scripts/list.js]: I don't know what this is supposed to be, but on my browser it shows up as one of those unknown character blocks?
image

File: miso-web/src/main/webapp/scripts/list.js [hunk 2]
```diff
@@ -298,6 +298,29 @@ ListUtils = (function($) {
...
+        "title": "Select all on all pages",
+        "handler": function() {
+          if (jqTable.fnSettings().fnRecordsDisplay() > 10000) {
+            Utils.showOkDialog('Select all', [ 'Too many items selected.', 'Please use stricter filtering.' ]);
```
Comment (avarsava) [inline miso-web/src/main/webapp/scripts/list.js]: Could we communicate the 10 000 item limit to users in the error message?

File: miso-web/src/main/webapp/scripts/list.js (hunk 1 above)
Comment (apmasell) [reply miso-web/src/main/webapp/scripts/list.js]: I � Unicode

File: miso-web/src/main/webapp/scripts/list.js (hunk 2 above)
Comment (apmasell) [reply miso-web/src/main/webapp/scripts/list.js]: Done.

**Comment by apmasell** [comment]
I talked to Ilinca and she doesn't really want to see all those things in the table, so this is what she's after.

**Comment by apmasell** [comment]
I mean, I can... :laughing:

**Pull request merged**